src/sdk/namespace_fs.js

```diff
@@ -1463,9 +1521,16 @@ class NamespaceFS {
     async _assign_versions_to_fs_xattr(fs_context, prev_ver_info, new_ver_stat, key, fs_xattr, delete_marker) {
         if (!prev_ver_info) prev_ver_info = await this.find_max_version_past(fs_context, key);
 
-        fs_xattr = Object.assign(fs_xattr || {}, {
-            [XATTR_VERSION_ID]: this._get_version_id_by_stat(new_ver_stat)
-        });
+        if (this._is_versioning_enabled()) {
+            fs_xattr = Object.assign(fs_xattr || {}, {
+                [XATTR_VERSION_ID]: this._get_version_id_by_stat(new_ver_stat)
+            });
+        } else if (this._is_versioning_suspended()) {
+            fs_xattr = Object.assign(fs_xattr || {}, {
+                [XATTR_VERSION_ID]: this._get_version_id_suspended()
```

Comment:
not sure if we must have it in a function but if so let's call something like get_version_null or get_null_version so we will know what to expect

Reply:
I change it to a const variable instead of a function and change its name.